nuxeo-jsf/nuxeo-platform-webapp-base/src/main/resources/OSGI-INF/layouts-contrib.xml

```diff
@@ -1910,6 +1927,10 @@
                   </widget>
                 </row>
                 <row>
+                  <widget category="widgetTypeConf">documentSuggestion_pageProviderPageSize
```

Comment:
You should add it to the edit mode of the `singleDocumentSuggestion` widget too

Reply:
and also in `select2Widget` and `select2WidgetMultiple` from nuxeo-features/nuxeo-platform-ui-select2/src/main/resources/OSGI-INF/widgettypes-contrib.xml

Reply:
Updated 👍